ts_src/psetv2/zkp.ts

```diff
@@ -438,7 +440,7 @@ export class ZKPGenerator {
     const { ecc } = this.zkp;
     const val = Buffer.alloc(32, 0);
     val.writeBigUInt64BE(BigInt(value), 24);
-    const result = ecc.privateMul(assetBlinder, val);
+    const result = Buffer.from(ecc.privateMul(assetBlinder, val));
     if (valueBlinder.length === 0) {
       throw new Error('Missing value blinder');
     }
```

Comment:
please, can we move this at the very beginning of the function to prevent weird behaviors of ecc APIs that are actually called before this check?